api/app/Http/Controllers/JobFormController.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\JobForm;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class JobFormController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('job-form.index',[
+            'forms' => JobForm::with('user')->get(),
+        ]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('job-form.create',[
+            'fields' => $this->getTableColumns('job_forms')->reject(function ($column) {
+                return in_array($column, ['id','created_at','updated_at', 'name', 'user_id']);
+            }),
+            'statuses' => DB::table('forms_field_status')->get(),
+        ]);
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     * @throws \Illuminate\Validation\ValidationException
+     */
+    public function store(Request $request)
+    {
+        $validators = $this->getValidators();
+        $attributes = $this->validate($request, $validators);
+
+        JobForm::create($attributes + ['user_id' => auth()->user()->id]);
+
+        return redirect()->back()->with('success', 'Job form created');
```

Comment:
I would prefer it redirected to job forms index instead of the same create page.

Reply:
Done